validator/graffiti/parse_graffiti.go

```diff
@@ -27,6 +38,45 @@ func ParseGraffitiFile(f string) (*Graffiti, error) {
 	if err := yaml.Unmarshal(yamlFile, g); err != nil {
 		return nil, err
 	}
+
+	specific := make(map[types.ValidatorIndex]string, len(g.Specific))
+	for i, o := range g.Specific {
+		specific[types.ValidatorIndex(i)] = ParseHexGraffiti(o)
+	}
+
+	g.Specific = specific
+	g.Default = ParseHexGraffiti(g.Default)
+	g.Ordered = parseGraffitiStrings(g.Ordered)
+	g.Random = parseGraffitiStrings(g.Random)
 	g.Hash = hashutil.Hash(yamlFile)
 	return g, nil
 }
+
+// ParseHexGraffiti checks if a graffiti input is being represented in hex and converts it to ASCII if so
+func ParseHexGraffiti(rawGraffiti string) string {
+	splitGraffiti := strings.SplitN(rawGraffiti, ":", 2)
+	if strings.ToLower(splitGraffiti[0]) == "hex" {
+		target := splitGraffiti[1]
+		if target == "" {
+			log.Debug("Blank hex tag to be interpreted as itself")
+			return rawGraffiti
+		}
+
+		if len(target) > 3 && target[:2] == "0x" {
+			target = target[2:]
+		}
+
+		if target == "" {
+			log.Debug("Nothing after 0x prefix, hex tag to be interpreted as itself")
```

Comment:
You can use WithField to show us what this graffiti is. It could be helpful for end users